docs/using-airbyte/getting-started/readme.md

```diff
@@ -1,43 +1,35 @@
 ---
 products: all
 ---
-
 # Airbyte Overview
 
 Airbyte is an open-source data integration platform designed to help you consolidate data from various sources into your data warehouses, lakes, and databases.
 
-Whether you're part of a large organization managing complex data pipelines or an individual analyst consolidating data for a personal project, Airbyte can work for you! We offer flexibility and scalability that's easy to tailor to your specific needs, from one-off jobs to enterprise-grade solutions. 
+Whether you're part of a large organization managing complex data pipelines or an individual analyst consolidating data for a personal project, Airbyte can work for you! We offer flexibility and scalability that's easy to tailor to your specific needs, from one-off jobs to enterprise-grade solutions.
+
 
-There are two options to consider to run Airbyte: 
-- **Airbyte Cloud** (recommended): allows you to focus on moving data while we take care of managing the infrastructure. 
-- **Self-Managed Airbyte**: can be deployed either locally or in an infrastructure you've set up. Our Self-Managed solution can be used as a free Community product or implemented with support of Enterprise operations. 
+## Airbyte Products
 
-:::tip
-If you have already deployed Airbyte or you've signed up to use Airbyte Cloud, and you're familiar with [Airbyte's core concepts](../../using-airbyte/core-concepts/), jump ahead to [Building Connections](../../cloud/managing-airbyte-cloud/configuring-connections.md).
-:::
+<Grid columns="2">
 
-## Airbyte Cloud
+<CardWithIcon title="Self-Managed Community (OSS)" description="Deploy locally or in your own infrastructure. Get started immediately without your data leaving your premises." ctaText="OSS Quickstart" ctaLink="oss-quickstart" icon="faDownload" />
```

Comment:
available icons are "fa-cloud", "fa-download", "fa-lock". I though it would be easier to use the name you can find in Fontawesome.

Reply:
It is! I as just being dumb, thanks!